tests/integration/conftest.py

```diff
@@ -18,6 +20,8 @@
 from trailblazer.store.models import Analysis
 from trailblazer.store.store import Store
 
+PIPELINES = [pipeline.lower() for pipeline in Pipeline]
```

Comment:
Can we move this as a classmethod in the pipeline class instead of a detached constant?

Reply:
It is only used in the tests, I don't want to add something in the source code that is only used in a test